bobtemplates/plone/addon/tox.ini

```diff
@@ -22,7 +22,7 @@ commands =
     {envbindir}/buildout -c {toxinidir}/{env:version_file} buildout:directory={envdir} buildout:develop={toxinidir} install test robot code-analysis
     coverage run {envbindir}/test -v1 --auto-color {posargs}
     # coverage run {envbindir}/test -v --all -t robot {posargs}
-    {envbindir}/code-analysis
+    {envbindir}/code-analysis {toxinidir}/src
```

Comment:
:+1:

Reply:
i removed this reacently completely, we need only tox for linting and testing. using both tools is confusing.